server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/DomainMappingsRoutes.java

```diff
@@ -182,14 +183,10 @@ public HaltException removeDomainMapping(Request request, Response response) thr
         return Optional.of(recipientRewriteTable.getStoredMappings(mappingSource).select(Mapping.Type.Domain))
             .filter(mappings -> mappings.contains(Mapping.Type.Domain))
             .map(this::toDomainList)
-            .orElseThrow(() -> ErrorResponder.builder()
-                .statusCode(HttpStatus.NOT_FOUND_404)
-                .type(ErrorResponder.ErrorType.NOT_FOUND)
-                .message(String.format("Cannot find mappings for %s", mappingSource.getFixedDomain()))
-                .haltError());
+            .orElse(ImmutableList.of());
```

Comment:
why would then entity (domainMapping) exist if there's no mapping?

Reply:
Because the associated domain exist?

Reply:
But that's not the requested resource. `/domainMappings/myDomain` does not exist.